packages/components/src/components/SquiggleEditor.tsx

```diff
@@ -21,9 +20,9 @@ export interface SquiggleEditorProps {
   /** If the result is a function, how many points along the function it samples */
   diagramCount?: number;
   /** The environment, other variables that were already declared */
-  environment?: exportEnv;
+  environment?: unknown;
```

Comment:
what is this? Is unknown a Rescript thing?

Reply:
`unknown` is all over the typescript files here.

Reply:
ah, forgot this was typescript

Reply:
It means that we don't know the type, and therefore can't do anything with it. I have it here because there is no environment in the reducer code until Umur's PR is merged